.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,7 @@
     "dockerfile": "Dockerfile"
   },
   "postCreateCommand": "bash -c 'if [[ $DISPLAY == *xquartz* ]]; then echo \"export DISPLAY=host.docker.internal:0\" >> /root/.bashrc; fi'",
```

Comment:
This doesn't work now, no permissions to edit roots bashrc

Reply:
can we run the devcontainer tests on macos too?